app/src/main/java/mozilla/lockbox/view/OnboardingAutofillFragment.kt

```diff
@@ -0,0 +1,32 @@
+package mozilla.lockbox.view
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.jakewharton.rxbinding2.view.clicks
+import io.reactivex.Observable
+import kotlinx.android.synthetic.main.fragment_autofill_onboarding.view.*
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import mozilla.lockbox.R
+import mozilla.lockbox.presenter.OnboardingAutofillPresenter
+import mozilla.lockbox.presenter.OnboardingAutofillView
+
+@ExperimentalCoroutinesApi
+class OnboardingAutofillFragment : Fragment(), OnboardingAutofillView {
+
+    override val onDismiss: Observable<Unit>
```

Comment:
naming nit: wrapping Observable named `onDismiss`, underlying button named `skipOnboarding`. Generally, these view observables are named things`skipClicks` to mirror the button they're hooked to (and for the following one, `enableClicks` or something similar).

Reply:
`onDismiss` -> `onSkipClick`
`onEnable` -> `onGoToSettingsClick`